modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1786 @@
+:description: Information about creating, listing, and dropping Neo4j's constraints.
+include::https://raw.githubusercontent.com/neo4j-graphacademy/courses/main/asciidoc/courses/cypher-indexes-constraints/ad.adoc[]
+= Create, show, and drop constraints
+
+This page describes how to create, list, and drop constraints.
+The following constraint types are available in Neo4j:
+
+* xref:constraints/managing-constraints.adoc#create-property-uniqueness-constraints[Property uniqueness constraints]
+* xref:constraints/managing-constraints.adoc#create-property-existence-constraints[Property existence constraints] label:enterprise-edition[]
+* xref:constraints/managing-constraints.adoc#create-property-type-constraints[Property type constraints] label:enterprise-edition[]
+* xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints] label:enterprise-edition[]
+
+
+[[create-constraint]]
+== CREATE CONSTRAINT
+
+Constraints are created with the `CREATE CONSTRAINT` command.
+When creating a constraint, it is recommended to provide a constraint name.
+This name must be unique among both indexes and constraints.
+If a name is not explicitly given, a unique name will be auto-generated.
+
+
+[NOTE]
+Creating a constraint requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-constraints[`CREATE CONSTRAINT` privilege].
+
+[NOTE]
+Adding constraints is an atomic operation that can take a while -- all existing data has to be scanned before a Neo4j DBMS can use a constraint.
+
+[[create-property-uniqueness-constraints]]
+=== Create property uniqueness constraints
+
+Property uniqueness constraints ensure that the property values are unique for all nodes with a specific label or all relationships with a specific type.
+For composite property uniqueness constraints on multiple properties, it is the combination of property values that must be unique.
+Property uniqueness constraints do not require all nodes or relationships to have values for the properties listed in the constraint.
+Only nodes or relationships that contain all properties specified in the constraint are subject to the uniqueness rule.
+Nodes or relationships missing one or more of the specified properties are not subject to this rule.
+Queries that try to add duplicated property values will fail.
```

Comment:
```suggestion
Property uniqueness constraints ensure that the property values are unique for all nodes with a specific label or all relationships with a specific type.
For composite property uniqueness constraints on multiple properties, it is the combination of property values that must be unique.
Queries that try to add duplicated property values will fail.

Property uniqueness constraints do not require all nodes or relationships to have values for the properties listed in the constraint.
Only nodes or relationships that contain all properties specified in the constraint are subject to the uniqueness rule.
Nodes or relationships missing one or more of the specified properties are not subject to this rule.
```
Maybe? It felt like a big block of text to read, and when I split it it felt like the last sentence belonged more with the first part than the section .